e2e-tests/cypress/e2e/pages/your-business/companies-house-search/companyDetails.js

```diff
@@ -0,0 +1,28 @@
+const companyDetails = {
+  errorSummary: () => cy.get('[data-cy="error-summary'),
```

Comment:
Could this just be in the E2E partials? Can see it's been added to `pages/shared` as well but it's already in partials?

Reply:
done